hack/gen-resources/cmd/commands/project.go

```diff
@@ -0,0 +1,59 @@
+package commands
+
+import (
+	"log"
+	"os"
+
+	"github.com/argoproj/argo-cd/v2/hack/gen-resources/tools"
+
+	generator "github.com/argoproj/argo-cd/v2/hack/gen-resources/generators"
+
+	"github.com/spf13/cobra"
+)
+
+func NewProjectCommand(opts *generator.GenerateOpts) *cobra.Command {
+	var command = &cobra.Command{
+		Use:   "project",
+		Short: "Manage applications",
```

Comment:
Should this be "Manage projects"?

Reply:
Agree, i will change that once back from vacation, thank you!